src/App.tsx

```diff
@@ -16,15 +20,22 @@ const TabsPage = () => {
 };
 */
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
 const App = () => (
   <div className="App">
-    <h1>Tabs with router</h1>
+    <h1 className="has-text-centered">Tabs with router</h1>
+    <Navigation />
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+      <Route path="tabs/:tabId" element={<TabsPage tabs={tabs} />} />
+      <Route path="/tabs" element={<TabsPage tabs={tabs} />} />
```

Comment:
use nested route, for more info – [here](https://reactrouter.com/docs/en/v6/getting-started/concepts#index-routes)